app/scripts/views/settings/communication_preferences.js

```diff
@@ -93,12 +94,23 @@ define(function (require, exports, module) {
     },
 
     setOptInStatus (newsletterId, isOptedIn) {
-      var method = isOptedIn ? 'optIn' : 'optOut';
+      let method, eventPrefix;
+      if (isOptedIn) {
+        method = 'optIn';
+        eventPrefix = '';
+      } else {
+        method = 'optOut';
+        eventPrefix = 'un';
+      }
+
       this.logViewEvent(method);
 
       return this.getMarketingEmailPrefs()[method](newsletterId)
         .then(() => {
           this.logViewEvent(method + '.success');
+          // Emit an additional flow event for consistency with
+          // the call to optIn in the account model
+          this.logFlowEvent(`newsletter.${eventPrefix}subscribed`);
```

Comment:
The reason we have two events is because `flow.newsletter.subscribed` is also emitted during sign-up if the user checked the box. Re-using the other event name there seemed like a bad option because it specifically mentions `settings.communication`.